<?xml version="1.0"?>

<launch>
    	<arg name="trajectory"  	default="0"/>
	<arg name="speed"  		default="5"/>
	<arg name="straight_speed" 	default="5"/>
	<arg name="x_d" 		default="1"/>
	<arg name="y_d" 		default="1"/>
	<arg name="z_d" 		default="1"/>
	<arg name="yaw_d" 		default="90"/>
	
	<node name="trajectory" pkg="wayspid" type="trajectory" output="screen" args="$(arg trajectory) $(arg speed) $(arg straight_speed) $(arg x_d) $(arg y_d) $(arg z_d) $(arg yaw_d)"/>
	<node name="dynamic_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" output="screen" args=""/>
</launch>

